src/Backend/Teradata/TeradataImportOptions.php

```diff
@@ -8,6 +8,10 @@
 
 class TeradataImportOptions extends ImportOptions
 {
+    public const CSV_ADAPTER_TPT = 1;
+    public const CSV_ADAPTER_SPT = 2;
+    private const DEFAULT_CSV_ADAPTER = self::CSV_ADAPTER_SPT;
```

Comment:
opravdu? to me privadi k otazce - proc chceme to STP vubec implementovat do driveru, kdyz jsme dosli k tomu, ze TPT nakonec asi funguje. Ja nechci zahodit tvoji praci, jen me zajima jakej od toho cekame vystup. Jako referencni mereni k TPT?

Reply:
JJ chtěl bych to trochu porovnat ten TPT (Fast load protocol) má nějaké omezení, které bactch import co dělá STP nemá, předpokládám, že to časem vyhodím, ale chtěl bych to ještě porovnat.

Reply:
Ale tady to asi prohodím a default bude TPT, každopádně je to pak default v driveru.

Reply:
ok, tak ci tak bych ale dal jako default TPT. ve suitach si to pak zapnes